test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 using Test
 using ExpressionTreeForge
-using JuMP, LinearAlgebra, MathOptInterface, ModelingToolkit, SparseArrays
+using JuMP, LinearAlgebra, MathOptInterface, SparseArrays
```

Comment:
First we list stdlib modules in alphabetical order.

Then we list external dependencies in alpha order.

The our own packages, also in alpha order.